Gqx/GJson.cpp

```diff
@@ -335,6 +335,16 @@ class GJsonPrivate : public QSharedData
       d( qstrdup( pSource.d ) )
     {}
 
```

Comment:
To the extent that we use a QMap, we do not need an assignment operator. Moreover, it can be successfully removed:
`Key& operator =(const Key& other) = delete;`